tests/bdd/features/volume/resize/test_resize_online.py

```diff
@@ -484,8 +484,8 @@ def the_new_capacity_should_be_available_for_the_application(test_volume_factory
 def the_failure_reason_should_be_invalid_arguments():
     """the failure reason should be invalid arguments."""
```

Comment:
Unrelated to this change - there is a commented function in this file `a published volume with more than one replicas` L290. Could you just delete that please?

Reply:
Removed this.